ludwig/cli.py

```diff
@@ -56,6 +56,7 @@ def __init__(self):
    init_config           Initialize a user config from a dataset and targets
    render_config         Renders the fully populated config with all defaults set
    check_install         Runs a quick training run on synthetic data to verify installation status
+   commit                Push trained model artifacts to Huggingface Hub
```

Comment:
Maybe to avoid a backward compatibility issue in the future, we can go ahead and make this either:

`ludwig commit_hf`

OR

`ludwig commit hf_hub`

Something like that. What do you think?

Reply:
See top-level comment, might make sense to change `commit` to `upload`.